docs/src/components/RouterLink/RouterLink.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './RouterLink.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkRouterLink = props => {
+  const classNames = [getClassName('bpkdocs-router-link')];
+  const { children, to, className, ...rest } = props;
+
+  if (className) {
+    classNames.push(className);
+  }
+
+  return (
+    <NavLink
+      className={classNames.join(' ')}
+      activeClassName={getClassName('bpkdocs-router-link--active')}
+      to={to}
+      {...rest}
+    >
+      {children}
```

Comment:
Why doesn't this render a BpkLink instead of using custom styles?

Reply:
https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md#component-reactcomponent

Reply:
In this case, FancyLink would be BpkLink

Reply:
It uses the `BpkLink` mixin behind the scenes. This is literally just copied from the existing component so I don't think it matters tbh

Reply:
Dogfooding is important though, and it'll save us changing this component if we change the behaviour of `BpkLink` in the future.

Reply:
Dammit why are you always right 😂

I'll change it, thanks for the feedback!